### PR TITLE
Make the tool to recognise command line flags

### PR DESCRIPTION
**Pull request by caggle**
So that only one of the tools could be run if needed.

Note that this implementation will probably be tweaked further when this issue: https://github.com/caggle/vautomator-standalone/issues/16 is addressed

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -9,9 +9,37 @@ all:
 build: Dockerfile docker-compose.yml
 	docker-compose build vautomator
 
-.PHONY: scan
-scan:
-	docker run -v ${PWD}/results:/app/results -it vautomator:latest ./run.py $(TARGET)
+.PHONY: force-build
```
Comment (gdestuynder) [inline Makefile]: maybe at this point we should actually use a wrapper rather than Makefile just for CLI
otherwise, this works ok :)

Comment (caggle) [reply Makefile]: I like the current approach. What would be an example wrapper be like?

File: lib/target.py [hunk 2]
```diff
@@ -109,16 +109,14 @@ def runTasks(self):
             elif isinstance(one_task, task.NessusTask):
                 nessus_results = one_task.runNessusScan()
                 if nessus_results:
-                    epoch_cdate = nessus_results.histories()[0].creation_date
+                    self.resultsdict.update({"nessus": True})
```
Comment (claudijd) [inline lib/target.py]: I feel like we're asking "Target" object to do too much here.  I  think we should consider another means to invoke specific scans without burying too much scanner type specific logic in here.

Comment (claudijd) [reply lib/target.py]: I feel like the purpose of having a class Structure is to help inform boundaries of where logic should live.   If we're burying a bunch  of task specific logic in Target, it just feels like we're blurring those lines too much.

File: run.py [hunk 3]
```diff
@@ -15,38 +16,135 @@
                     datefmt="%Y-%m-%d %I:%M:%S %p %Z")
 
 
-def setupVA(va_target):
+def parseCmdArgs():
```
Comment (claudijd) [inline run.py]: We're missing unit-tests for all of this logic.  I'd recommend making the  command parsing it's own class to condense this and make it testable.

File: Makefile (hunk 1 above)
Comment (gdestuynder) [reply Makefile]: if you like it you should keep it

A wrapper would be "vaautod.py" such as 
```
#!/bin/bash
docker run -v ${PWD}/results:/app/results -it vautomator:latest ./run.py $*
```
The difference is that it allows for multiple arguments in one go without doing `make ARGS="arg1 arg2" TARGET="lala.com" scan` (lots to type) or `make arg TARGET="lala.com"` (2nd one can't take more args)

**Pull request merged**